README.md

```diff
@@ -79,6 +81,9 @@ The value will be interpreted as time unit defined in _cleanup-time-unit_ settin
 #### cleanup-time-unit
 The time unit for _cleanup-older-than_ setting. Available values are MILLIS, SECONDS, MINUTES, HOURS. _The default value is SECONDS_
 
+#### max-event-batch-content-size
+The max size in bytes of summary events content in a batch defined in _max-event-batch-content-size_ setting. _The default value is 1048576_
+
 ## Requried pins
 
 The Check1 component has got tow types of pin
```

Comment:
```suggestion
The Check1 component will have two types of pin:
```

Reply:
I think the `The Check1 component has two types of pin:` sentence will be more applicable for this case.